src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -114,15 +115,31 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
     StringBuilder builder = new StringBuilder();
     builder.append("{");
 
-    String userEmail = "unknown";
-    BasicCommand getGitUser = new BasicCommand("git", "config", "--global", "user.email");
-    getGitUser.run();
-    if (getGitUser.exitValue() == Command.SUCCESS) {
-      userEmail = getGitUser.getOutput().trim();
+    String userId = "unknown";
+
+    if (GatorGradleConfig.get().hasIdCommand() == true) {
+      BasicCommand getUserId = null;
+      if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).equals("windows")) {
```

Comment:
Use `GatorGradlePlugin.OS` and `GatorGradlePlugin.WINDOWS` to test here -- see https://github.com/GatorEducator/gatorgradle/blob/master/src/main/java/org/gatorgradle/GatorGradlePlugin.java#L50

Reply:
Fixed in https://github.com/GatorEducator/gatorgradle/pull/71/commits/1ea05fd470ed3a8b6472f2a12656ef50cda8020d